builtin/providers/google/resource_google_project.go

```diff
@@ -31,18 +32,29 @@ func resourceGoogleProject() *schema.Resource {
 		Delete: resourceGoogleProjectDelete,
 
 		Schema: map[string]*schema.Schema{
-			"id": &schema.Schema{
+			"project_id": &schema.Schema{
```

Comment:
This is a backwards-incompatible change, isn't it? Do we need a state migration here?